backend-api/Dockerfile.test

```diff
@@ -0,0 +1,26 @@
+FROM node:iron-alpine AS builder
```

Comment:
question - is `iron` general knowledge as a major version of Node? 

I'm wondering whether it'd be clearer to people looking at these files to reference the version number instead as the base image?

Reply:
Separately - do we need a builder here - it looks unused

Reply:
Ah yes, i started with builder but it didn't seem to be doing anything which wouldn't be brought
over to the final image. I have remove that. 


Re: `iron`, I matched the value from `.nvmrc`. I would change both if i was to change it here.

Reply:
`iron` - understood :) Happy to keep with what you've done